### ISSUE TITLE
RGI05: Flade Isblink divides

### ISSUE THREAD

**Issue by fmaussion** [GLIMS-RGI/rgi7_scripts#39]
Flade Isblink is the largest entity in RGI6 (by far) and therefore also in RGI7. The reason is that it is not divided:

![image](https://user-images.githubusercontent.com/10050469/149838487-878af491-a960-4054-aefe-c756abfffcb9.png)

After some emails back and forth with Marco Möller and Frank Paul, here is Frank's summary:

**A) the divides shape file**

1. what phil has sent you are seemingly the raw calculated drainage basins, they need to be properly merged to represent real glaciers, this is some work
2. In the south the dataset seems incomplete and cut, likely due to missing DEM data
3. the divides might have been calculated with the GIMP DEM, this needs to be recalculated using the ArcticDEM
=> the work to be done is thus - download, resample, mosaic the ArcticDEM, calculate divides based on a flow direction grid, merge basins, intersect with outlines

**B) divide it or not**

The reason why some ice caps are divided while others are not is related to the rule set applied back then. In principle, we left ice caps with missing outlet glaciers undivided. Although we can make an exception for Flade Isblink (if time to conduct the steps listed under point A is around), but we prefer to not extent this exercise to other yet undivided ice caps. The main reason is the inconsistency with other ice caps on Greenland, i.e. it has to be done with all of them rather a few selected ones. If manual adjustments are required, this is a considerable effort (several person months) rather than a background activity.

=> We will check with Fabien temporal possibilities for the Flade Isblink adjustment.

I'm leaving this here in case someone wants to tackle this for RGI7.0 or not.

**Comment by fmaussion** [comment]
Here is the file that Marco Möller sent me: 
[refladeisblinkoutlines.zip](https://github.com/GLIMS-RGI/rgi7_scripts/files/10517804/refladeisblinkoutlines.zip)

Here is what is in GLIMS/RGI right now: 
[sel_FladeIsbkink_GLIMS.zip](https://github.com/GLIMS-RGI/rgi7_scripts/files/10517887/sel_FladeIsbkink_GLIMS.zip)

After visual inspection I see indeed the issues that Frank was referring to. There is some work needed to merge the two. The divides files needs some manual intervention to remove sliver polygons and artifacts, and/or this information needs to be used to divide the original outlines (Marco/Phil's file cant be used as replacement). 

I can see that this represents more work than I thought.

**Comment by bearecinos** [comment]
I'm posting here a nicer picture of the many problems @fmaussion 
![image](https://user-images.githubusercontent.com/16323300/215091931-a5009643-18ef-4ff2-b388-189fa1373592.png)

In this [Qgis-Greenland](https://qgreenland.org/download) there is some data from ice basins ... Maybe that could help. 
Adding ice basins:
![image](https://user-images.githubusercontent.com/16323300/215092388-b854dccb-5bf4-4454-8ada-0149e8462430.png)

**Comment by iamdonovan** [comment]
I spent some time on this over the weekend, cleaning up the divides and also some of the other issues (invalid geometries, random holes in the polygons, etc). I don't have a pretty screenshot to post from my office computer, but I'll continue working on it and should hopefully have it done by Wednesday.

**Comment by fmaussion** [comment]
Thanks @bearecinos! For glims the attributes does not yet play a role. We will try to add attributes such as tidewater over the next weeks based on RGI7 (email will follow about this).

For now, a divide of the one big outline (or the group of glaciers, depending in how Bob is doing it) and a resubmission to GLIMS would be a big step forward for sure.

**Comment by iamdonovan** [comment]
Attached a screenshot showing my progress so far - I've added divides for most of the largest outline, and worked on cleaning up some of the boundaries to match the image a bit better, plus a bit of simplifying the outlines.

For the divides, I used a combination of the ArcticDEM 10m and Copernicus 30m DEM hillshades, the file provided by Marco/Bea, and the watersheds output through the SAGA Channel Network and Drainage Basins tool.

I will spend a bit more time on this tomorrow by dividing the second-largest ice cap (RGI60-05.10148), and maybe do a little bit more to clean up some of the interior boundaries, before sending the file along.

![Screenshot from 2023-01-31 00-39-34](https://user-images.githubusercontent.com/5552374/215629012-afe74f94-363f-453e-8dc2-40fe3ba1e660.png)
